# Human Microbiome Analysis Guide

[![Check Resources](https://github.com/erictleung/microbiome-analysis-resources/actions/workflows/test.yml/badge.svg)](https://github.com/erictleung/microbiome-analysis-resources/actions/workflows/test.yml)

Guides on analyzing the human microbiome.

**Table of Contents**

- [Statistical References](#statistical-references)
- [Tutorials and Online Courses](#tutorials-and-online-courses)
- [Software](#software)
- [Journal Clubs](#journal-clubs)
- [Topic and Collections](#topic-and-collections)
- [License](#license)

## Statistical References

- [Guide to Statistical Analysis in Microbial Ecology (GUSTA ME)][gustame]
- [Ordination Methods for Ecologists][ordination]

[gustame]: http://mb3is.megx.net/gustame
[ordination]: http://ordination.okstate.edu/

## Tutorials and Online Courses

- [Bioconductor Workflow for Microbiome Data Analysis: from raw reads to
  community analyses][f1000]
- [Microbiome Discovery with Dan Knights][dknights]
- [Riffomonas Reproducible Research Tutorial Series][riff]
- [SISMID Microbiome Data Analysis 2017][sismid2017]
- [SISMID Microbiome Data Analysis 2018][sismid2018]
- [Open & Reproducible Microbiome Data Analysis Spring School 2018][openmicrobiome]
- [Amplicon Analysis - AstrobioMike][abm]
- [Strategies and Techniques for Analyzing Microbial Population Structures
  (STAMPS) (2019)][stamps2019]
- [16S rRNA analysis by Rachael Lappan][16srachael]
- [Dr. Rosen's Computational Metagenomics Special Topics Course][rosenmicro]

[f1000]: http://dx.doi.org/10.12688/f1000research.8986.2
[dknights]: https://www.youtube.com/playlist?list=PLOPiWVjg6aTzsA53N19YqJQeZpSCH9QPc
[riff]: https://www.riffomonas.org/reproducible_research/
[sismid2017]: http://dev-biostat.pantheonsite.io/suminst/archives/SISMID2017/MD1706
[sismid2018]: http://www.biostat.washington.edu/suminst/archives/SISMID2018/MD1808
[openmicrobiome]: https://mibwurrepo.github.io/Microbial-bioinformatics-introductory-course-Material-2018/introduction.html
[abm]: https://astrobiomike.github.io/amplicon/
[stamps2019]: https://github.com/mblstamps/stamps2019
[16srachael]: https://rachaellappan.github.io/16S-analysis/
[rosenmicro]: https://github.com/EESI/Comp_Metagenomics_resources

## Software

- [Advancing our understanding of the human microbiome using
  QIIME][navas-molina] - Overview of the motivation for the well-known analysis
  package, QIIME, and an entire workflow from preprocessing to diversity
  analysis and more. Highlights 16S rRNA sequencing analysis.

[navas-molina]: https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC4517945

## Journal Clubs

- [META Center for Systems Biology][uo] - Journal club at the University of
  Oregon Microbial Ecology and Theory of Animals (META) Center.
- [McMaster Human Microbiome Journal Club][mcmast] - Journal club at the
  McMaster University in Ontario, Canada.

[uo]: http://meta.uoregon.edu/journal-club/
[mcmast]: https://hmjournalclub.wordpress.com/

## Topic and Collections

- [Science Magazine][sciencemag]
- [PLoS The Human Microbiome Project][ploshmp]
- [The Microbiome and Epidemiology][annalsepi]

[sciencemag]: http://www.sciencemag.org/topic/microbiome
[ploshmp]: http://collections.plos.org/hmp
[annalsepi]: http://www.sciencedirect.com/science/journal/10472797/26/5

## License

[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)

To the extent possible under law, [Eric Leung](https://erictleung.com) has
waived all copyright and related or neighboring rights to this work.
